tests/Browser/BlogManageTest.php

```diff
@@ -0,0 +1,562 @@
+                    ->waitFor('#createModal')
+                    ->type('title', $textOver100)
+                    ->script("CKEDITOR.instances['blog-content'].setData('content');");
+
+            $browser->waitForText('Submit')
+                    ->press('Submit')
+                    ->waitForLocation($data['path']['blog'])
+                    ->assertSee('The title may not be greater than 100 characters.')
+                    ->assertDontSee($textOver100);
+        });
+    }
+
+    /**
+     * Test create blog with user can see blog.
+     * case 2.4: Create blog: Do not fill input
+     * Error
+     *
+     * @return void
+     */
+    public function testCreateBlogDoNotFillInput()
+    {
+        $user = $this->userCanSee();
+        $data = $this->getDataTest();
+        $this->browse(function ($browser) use ($user, $data) {
+            $browser->visit($data['path']['login'])
+                    ->loginAs($user)
+                    ->visit($data['path']['blog'])
+                    ->press('Create Blog')
+                    ->waitFor('#createModal')
+                    ->waitForText('Submit')
+                    ->press('Submit')
+                    ->waitForLocation($data['path']['blog'])
+                    ->assertSee('The title field is required.')
+                    ->assertSee('The content field is required.');
+        });
+    }
+
+    /**
+     * Test create blog with user can see blog.
+     * case 2.5: - Create blog: Content input empty
+     * Error
+     *
+     * @return void
+     */
+    public function testCreateBlogContentInputEmpty()
+    {
+        $user = $this->userCanSee();
+        $data = $this->getDataTest();
+        $this->browse(function ($browser) use ($user, $data) {
+            $browser->visit($data['path']['login'])
+                    ->loginAs($user)
+                    ->visit($data['path']['blog'])
+                    ->press('Create Blog')
+                    ->waitFor('#createModal')
+                    ->type('title', 'new title')
+                    ->waitForText('Submit')
+                    ->press('Submit')
+                    ->waitForLocation($data['path']['blog'])
+                    ->assertSee('The content field is required.');
+        });
+    }
+
+    /**
+     * Test create blog with user can see blog.
+     * case 2.6: - Create blog: Title input empty
+     * Error
+     *
+     * @return void
+     */
+    public function testCreateBlogTitleInputEmpty()
+    {
+        $user = $this->userCanSee();
+        $data = $this->getDataTest();
+        $this->browse(function ($browser) use ($user, $data) {
+            $browser->visit($data['path']['login'])
+                    ->loginAs($user)
+                    ->visit($data['path']['blog'])
+                    ->press('Create Blog')
+                    ->waitFor('#createModal')
+                    ->script("CKEDITOR.instances['blog-content'].setData('content');");
```

Comment:
why there are 2 style related to empty , 1 without code like this once , 1 with code like L363

Reply:
I fixed